sdk/containerregistry/Azure.Containers.ContainerRegistry/src/Models/ArtifactManifestProperties.cs

```diff
@@ -17,13 +17,13 @@ public ArtifactManifestProperties()
         {
         }
 
-        /// <summary> Delete enabled. </summary>
+        /// <summary> Whether or not this tag can be deleted. </summary>
         public bool? CanDelete { get; set; }
-        /// <summary> Write enabled. </summary>
+        /// <summary> Whether or not this tag can be written to. </summary>
         public bool? CanWrite { get; set;  }
-        /// <summary> List enabled. </summary>
+        /// <summary> Whether or not to include this tag in the collection returned from the tag listing operation. </summary>
```

Comment:
Is there a particular method that we can reference here that lists the tags?

Reply:
Yes, that I can add.